### PR TITLE
[BrokenLinksH2]

### PR DESCRIPTION
**Pull request by mohitp930**

### CODE REVIEW

**Comment by mohitp930** [comment]
Global effort to fix broken links

@mikehoffms

[NOTE] What's New in DevTools (Microsoft Edge 81) has its own section on the now discontinued extension. I've opted to fix the broken link by linking to Microsoft Edge Tools for VS Code. But a large editorial change is likely needed, which is outside of my scope. I'd appreciate any feedback you have.

The Content & Learning team is fixing broken links on docs.microsoft.com for the rest of H2. This effort will eliminate potential accessibility, security, and usability issues. This PR includes only link fixes and does not change other content. Please review within five business days and merge, or comment in the PR with any changes you'd like to see. Thanks!

File: microsoft-edge/devtools-guide-chromium/whats-new/2020/01/devtools.md [hunk 1]
```diff
@@ -111,7 +111,7 @@ For more information, check out [Elements for Microsoft Edge Visual Studio Code
 
 #### Debugger for Microsoft Edge
 
-With the [Debugger for Microsoft Edge](https://marketplace.visualstudio.com/items?itemName=msjsdiag.debugger-for-edge) Visual Studio Code extension, debug JavaScript running in Microsoft Edge directly from Visual Studio Code.
+With the [Microsoft Edge Tools for VS Code](https://marketplace.visualstudio.com/items?itemName=ms-edgedevtools.vscode-edge-devtools) Visual Studio Code extension, debug JavaScript running in Microsoft Edge directly from Visual Studio Code.
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/whats-new/2020/01/devtools.md:114]: @codepo8 - https://marketplace.visualstudio.com/items?itemName=msjsdiag.debugger-for-edge has become 404.  Should this h4 section be removed? Rendered public What's New entry has the 404 link:  https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/whats-new/2020/01/devtools#debugger-for-microsoft-edge

Is this page valid? - https://docs.microsoft.com/en-us/microsoft-edge/visual-studio-code/debugger-for-edge

I'm working in an "Old What's New processing" PR https://github.com/MicrosoftDocs/edge-developer/pull/1881 where I can do more elaborate fixes.  The present PR could just delete the 404 link or wrap with backticks to disable it.

Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/whats-new/2020/01/devtools.md:114]: @codepo8 - I'm closing this Comment, to approve this PR, to fix the 404 asap.  We can continue discussing redesign in my "Old What's New" processing PR 1881, re: what to do with old mentions of Debugger Extension.  Probably all mentions of the Debugger Extension are in old What's New items.

**Review by mikehoffms — APPROVED** [review]
Thanks.  Consulting with SME on how to handle mentions of old tech in old What's New items, in other PR.  I think these fixes are suitable solutions for old What's New entries.

**Pull request merged**